src/main/java/reactor/core/publisher/TopicProcessor.java

```diff
@@ -229,18 +392,18 @@
 	 * buffer
 	 * @param <E> Type of processed signals
 	 * @return a fresh processor
+	 * @deprecated use {@link Builder#build()}
 	 */
+	@Deprecated
 	public static <E> TopicProcessor<E> create(String name, int bufferSize,
 	                                                WaitStrategy strategy,
 	                                                Supplier<E> signalSupplier) {
-		return new TopicProcessor<>(name,
-				bufferSize,
-				strategy == null ?
```

Comment:
to be entirely foolproof and consistent, the builder should not be returned directly but be configured in a few steps, including a check for `strategy == null` before chaining in `.waitStrategy(strategy)`.